src/View/mainpage/DrawROIWindow/UIDrawROIWindow.py

```diff
@@ -1132,6 +1130,45 @@ def onBoxDrawClicked(self):
         self.disable_cursor_diameter_transparency()
         self.has_drawing = False
 
+    def onClearClicked(self):
```

Comment:
I would recommend running flake8 on this file to meet the styling requirements. this can be done in the cmd prompt by running ```flake8 src/View/mainpage/DrawROIWindow/UIDrawROIWindow.py```. As a specific example, the function name should be snake_case: on_clear_clicked

Reply:
To be honest I tried before, but this will come up: qt.core.qmetaobject.connectslotsbyname: QMetaObject::connectSlotsByName: No matching signal for on_clear_clicked(). I have to follow the signature style to be used with connectSlotsByName instead of snake_case. More info here: https://forum.qt.io/topic/32092/no-matching-signal-for-solved

Reply:
Qt is known to have stuck with CamelCase.
Its one of those things that one has to live with...

Reply:
Ah okay i see, makes sense then